--- origsrc/lbreakout2-2.6.2/Makefile.am	2010-10-04 08:41:03.000000000 -0500
+++ src/lbreakout2-2.6.2/Makefile.am	2010-11-04 00:46:30.120713200 -0500
@@ -21,7 +21,7 @@ AUTOMAKE_OPTIONS = foreign
 # does not contain plus signs. :-)
 lbreakout2.desktop:
 	@sed -e "s+Icon=+Icon=$(datadir)/icons/lbreakout48.gif+" \
-	< lbreakout2.desktop.in > lbreakout2.desktop
+	< $(srcdir)/lbreakout2.desktop.in > lbreakout2.desktop
 
 win32text: $(WIN32_TEXT)
 	@echo Converting text files for win32 installer
--- origsrc/lbreakout2-2.6.2/client/Makefile.am	2010-11-04 00:47:21.584260700 -0500
+++ src/lbreakout2-2.6.2/client/Makefile.am	2010-11-04 00:34:53.359479200 -0500
@@ -51,7 +51,7 @@ install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)
 	$(mkinstalldirs) $(DESTDIR)$(hi_dir)
 	if ! test -f $(DESTDIR)$(hi_dir)/lbreakout2.hscr; then \
-	  $(INSTALL_DATA) -m 666 empty.hscr $(DESTDIR)$(hi_dir)/lbreakout2.hscr; \
+	  $(INSTALL_DATA) -m 666 $(srcdir)/empty.hscr $(DESTDIR)$(hi_dir)/lbreakout2.hscr; \
 	fi;
 
 uninstall-local:
--- origsrc/lbreakout2-2.6.2/client/gfx/AbsoluteB/Makefile.am	2010-11-04 00:47:21.589261000 -0500
+++ src/lbreakout2-2.6.2/client/gfx/AbsoluteB/Makefile.am	2010-11-04 00:34:53.371479900 -0500
@@ -6,7 +6,7 @@ ABOUT README
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/AbsoluteB
 	@for file in $(EXTRA_DIST); do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/AbsoluteB/$$file;\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/AbsoluteB/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/gfx/Classic/Makefile.am	2010-11-04 00:47:21.594261300 -0500
+++ src/lbreakout2-2.6.2/client/gfx/Classic/Makefile.am	2010-11-04 00:34:53.377480200 -0500
@@ -3,7 +3,7 @@ EXTRA_DIST = ABOUT
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/Classic
 	@for file in $(EXTRA_DIST); do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/Classic/$$file;\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/Classic/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/gfx/Makefile.am	2010-11-04 00:47:21.598261500 -0500
+++ src/lbreakout2-2.6.2/client/gfx/Makefile.am	2010-11-04 00:34:53.365479600 -0500
@@ -19,7 +19,7 @@ SUBDIRS = AbsoluteB Oz Moiree Classic
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx
 	@for file in $(EXTRA_DIST); do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/$$file;\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/gfx/Moiree/Makefile.am	2010-11-04 00:47:21.603261800 -0500
+++ src/lbreakout2-2.6.2/client/gfx/Moiree/Makefile.am	2010-11-04 00:34:53.382480500 -0500
@@ -7,7 +7,7 @@ ABOUT
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/Moiree
 	@for file in $(EXTRA_DIST); do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/Moiree/$$file;\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/Moiree/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/gfx/Oz/Makefile.am	2010-11-04 00:47:21.607262000 -0500
+++ src/lbreakout2-2.6.2/client/gfx/Oz/Makefile.am	2010-11-04 00:34:53.388480900 -0500
@@ -6,7 +6,7 @@ README
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/Oz
 	@for file in $(EXTRA_DIST); do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/Oz/$$file;\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/Oz/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/gui_theme/Makefile.am	2010-11-04 00:47:21.612262300 -0500
+++ src/lbreakout2-2.6.2/client/gui_theme/Makefile.am	2010-11-04 00:34:53.393481200 -0500
@@ -9,7 +9,7 @@ edit.wav click.wav
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gui_theme
 	@for file in $(EXTRA_DIST); do \
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gui_theme/$$file; \
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gui_theme/$$file; \
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/levels/Makefile.am	2010-11-04 00:47:21.617262600 -0500
+++ src/lbreakout2-2.6.2/client/levels/Makefile.am	2010-11-04 00:34:53.398481400 -0500
@@ -115,7 +115,7 @@ Zufallswelt
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/levels
 	@for file in $(EXTRA_DIST); do \
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/levels/$$file; \
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/levels/$$file; \
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/client/sounds/Makefile.am	2010-11-04 00:47:21.625263000 -0500
+++ src/lbreakout2-2.6.2/client/sounds/Makefile.am	2010-11-04 00:34:53.404481800 -0500
@@ -8,8 +8,8 @@ bonus_magnet.wav malus_magnet.wav disabl
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/sounds
-	@for file in *.wav; do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/sounds/$$file;\
+	@for file in $(EXTRA_DIST); do\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/sounds/$$file;\
 	done
 
 win32-install-script:
--- origsrc/lbreakout2-2.6.2/docs/Makefile.am	2010-11-04 00:47:21.629263300 -0500
+++ src/lbreakout2-2.6.2/docs/Makefile.am	2010-11-04 00:34:53.409482100 -0500
@@ -8,10 +8,9 @@ warp.jpg screen.jpg
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(doc_dir)/lbreakout2
-	@for file in *.jpg; do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(doc_dir)/lbreakout2/$$file;\
+	@for file in $(EXTRA_DIST); do\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(doc_dir)/lbreakout2/$$file;\
 	done
-	$(INSTALL_DATA) index.html $(DESTDIR)$(doc_dir)/lbreakout2/index.html
 
 win32-install-script:
 	@for file in $(EXTRA_DIST); do \
--- origsrc/lbreakout2-2.6.2/server/levels/Makefile.am	2010-11-04 00:47:21.634263500 -0500
+++ src/lbreakout2-2.6.2/server/levels/Makefile.am	2010-11-04 00:34:53.419482600 -0500
@@ -3,7 +3,7 @@ EXTRA_DIST = N_Arenas N_Snapshots
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/levels
 	@for file in $(EXTRA_DIST); do \
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/levels/$$file; \
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/levels/$$file; \
 	done
 
 win32-install-script:
